lib/debts/letter_downloader.rb

```diff
@@ -35,8 +35,9 @@ def list_letters
       end
 
       debts_records.map do |debts_record|
-        debts_record.marshal_dump.slice(
-          :document_id, :doc_type, :type_description, :received_at
+        record = debts_record.marshal_dump.deep_transform_keys { |key| key.to_s.camelize(:lower) }
```

Comment:
Could you use a serializer instead? Then I think olive branch will take care of the camel casing if the X-Key-Inflection header is set to camel.